src/main/java/seedu/address/model/book/BookList.java

```diff
@@ -80,6 +80,27 @@ public void setBooks(List<Book> books) {
         internalList.setAll(books);
     }
 
+    /**
+     * Replaces all books borrowed by {@code borrower} with the same book, but with available status in this book list.
+     */
+    public void returnAllBorrowedBooks(Patron borrower) {
+        for (Book book : internalList) {
+            if (!book.isBorrowedBy(borrower)) {
+                continue;
+            }
+            Book updatedAvailableBook = new Book(book, BookStatus.createAvailableBookStatus());
+            setBook(book, updatedAvailableBook);
+        }
+    }
```

Comment:
Good method to change all borrowed books to available! The method is simple to understand and not deeply nested :100: